jenkins-plugin/pom.xml

```diff
@@ -87,6 +87,66 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <!-- we need to force some dependencies introduced by plugin depending on maven-plugin such jacoco or invoker-->
```

Comment:
Why are those `<dependencies>` and not `<dependencyManagement>`?

Reply:
This will not work with enforcer-plugin/RequireUpperBoundDeps and those dependencies version need to be enforced.

Reply:
So this means that the mechanism to build jenkins plugins requires to list dependencies that are in fact not used directly by the plugin itself? 

This means that we have to declare "commons-digester" as a dependency even if the code of the pipeline-maven-plugin does not directly reference any class of "commons-digester"?

Reply:
mostly it is (because of the enforcer rule).
But I will change those dependencies as optional to make enforcer happy.....